documentation/quarkus/guide-building-native-image.asciidoc

```diff
@@ -0,0 +1,35 @@
+:toc: macro
+toc::[]
+
+= Building a native image
+
+Quarkus provides the ability to create a native executable of the application. This improves the startup time, performance and efficiency. Also, a JVM is not needed because it is included in the application. A distribution of GraalVM is needed. You can find the differences between the available distributions https://quarkus.io/guides/building-native-image#graalvm[here].
+
+=== Installing GraalVM
+
+A native image can be created locally or through a container environment. To create a native image locally an installed and configured version of GraalVM is needed, you can follow the https://quarkus.io/guides/building-native-image#prerequisites-for-oracle-graalvm-ceee[installation guide from quarkus] or the https://www.graalvm.org/docs/getting-started/#install-graalvm[guide provided by GraalVM] for this. 
+
+
+
+=== Build a native executable with GraalVM
+
+A working quarkus application is needed, to build a quarkus application you can read the link:quarkus-template.asciidoc#basic-templates[basic Quarkus application chapter] 
```

Comment:
Do you think we really need this part? I think if someone reads the guide to create the native image, the application is already there